### PR TITLE
dummy input for graph traversal

### PR DESCRIPTION
**Pull request by ryanchou-dev**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

File: content/3_Silver/Graph_Traversal.mdx [hunk 1]
```diff
@@ -380,7 +405,13 @@ public static void main(String[] args) throws IOException {
...
-	// Define adjacency list and read in problem-specific input
+	/*
+	 * Define adjacency list and read in problem-specific input
+	 *
+	 * In this example, we've provided "dummy input" that's
+	 * reflected in the GIF above to help illustrate the
+	 * order of the recrusive calls.
+	 */
 
 	for (int i = 0; i < MAX_N; i++) {
```
Comment (ryanchou-dev) [inline content/3_Silver/Graph_Traversal.mdx:416]: @SansPapyrus683

if this is okay with you, could you also make the changes to the java code?

File: content/3_Silver/Graph_Traversal.mdx [hunk 2]
```diff
@@ -151,9 +167,9 @@ static void dfs(int currentNode) {
 <CPPSection>
 
 ```cpp
-const int MAX_N = 1e6;
-vector<int> adj[MAX_N];
-bool visited[MAX_N];
+int n = 6;
+vector<int> adj[n];
+bool visited[n];
```
Comment (ChuangSheep) [inline content/3_Silver/Graph_Traversal.mdx:172]: variable length array is not part of c++ standard. Use `const` instead.

File: content/3_Silver/Graph_Traversal.mdx [hunk 3]
```diff
@@ -407,15 +438,22 @@ public static void main(String[] args) throws IOException {
 <CPPSection>
 
 ```cpp
-const int MAX_N = 1e6;
+int n = 6;
+vector<int> adj[n];
+bool visited[n];
```
Comment (ChuangSheep) [inline content/3_Silver/Graph_Traversal.mdx:443]: s. a.

File: content/3_Silver/Graph_Traversal.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply content/3_Silver/Graph_Traversal.mdx:172]: or just use a vector

**Review by SansPapyrus683 — APPROVED** [review]
alr everything should compile, maybe yall should test on your own machines as well

File: content/3_Silver/Graph_Traversal.mdx (hunk 3 above)
Comment (ChuangSheep) [reply content/3_Silver/Graph_Traversal.mdx:453]: this is not fixed yet - VLA is not allowed in c++

Comment (ryanchou-dev) [reply content/3_Silver/Graph_Traversal.mdx:453]: yepp im aware -- fixing it now

**Review by ChuangSheep — APPROVED** [review]
alright, though i find an array of ArrayList in Java really sus
~Yea it compiles but just weird for java~

**Pull request merged**